diff -upr trojan-1.15.1/src/core/service.cpp trojan-1.15.1.new/src/core/service.cpp
--- trojan-1.15.1/src/core/service.cpp	2020-03-25 13:22:14.000000000 +0800
+++ trojan-1.15.1.new/src/core/service.cpp	2020-06-01 16:08:23.380000000 +0800
@@ -229,7 +229,7 @@ Service::Service(Config &config, bool te
         }
     }
     if (config.ssl.cipher != "") {
-        SSL_CTX_set_cipher_list(native_context, config.ssl.cipher.c_str());
+        SSL_CTX_set_cipher_list(native_context, "PROFILE=SYSTEM");
     }
     if (config.ssl.cipher_tls13 != "") {
 #ifdef ENABLE_TLS13_CIPHERSUITES
